php/plan.sh

```diff
@@ -0,0 +1,68 @@
+pkg_name=php
+pkg_distname=php
+pkg_origin=core
+pkg_version=7.1.2
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('PHP-3.01')
+pkg_upstream_url=http://php.net/
+pkg_description="PHP is a popular general-purpose scripting language that is especially suited to web development."
+pkg_source=https://php.net/get/${pkg_distname}-${pkg_version}.tar.bz2/from/this/mirror
+pkg_filename=${pkg_distname}-${pkg_version}.tar.bz2
+pkg_dirname=${pkg_distname}-${pkg_version}
+pkg_shasum=e0f2214e2366434ee231156ba70cfefd0c59790f050d8727a3f1dc2affa67004
+pkg_deps=(
+  core/coreutils
+  core/curl
+  core/glibc
+  core/libxml2
+  core/libjpeg-turbo
+  core/libpng
+  core/openssl
+  core/zlib
+)
+pkg_build_deps=(
+  core/bison2
+  core/gcc
+  core/m4
```

Comment:
`m4` is needed by `bison` to generate the parser…
ubuntu's bison depends on `m4`

```
Depends: m4, libc6 (>= 2.15), libbison-dev (= 2:3.0.4.dfsg-1)
```
as does alpine's: https://pkgs.alpinelinux.org/package/v3.5/main/x86_64/bison
and archlinux: https://www.archlinux.org/packages/core/x86_64/bison/